src/modules/tiles/TilePalettePanel.hx

```diff
@@ -267,6 +267,14 @@ class TilePalettePanel extends SidePanel
 
 		if (tileset != null)
 		{
+			if (image.width <= 0)
+			{
+				image.addEventListener("load", function (e) {
```

Comment:
Do we need to remove the listener here once we're done with it? or is that something that gets done automatically?

Reply:
Yeah a dangling callback could blow up eventually. Fixed in new commit 👍